app/components/school_recruited_transition_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= govuk_notification_banner(title: "Important") do |banner| %>
+  <% banner.add_heading(text: "#{delivery_partner.name} has confirmed they will be delivering an induction programme to your early career teachers, starting in #{cohort.display_name}.") %>
+  <%# TODO: Add a correct link! %>
+  <p>If this is a mistake, <%= govuk_link_to "report it now", challenge_partnership_path(partnership: partnership.id) %></p>
+  <p>If you do not report this by <%= partnership.challenge_deadline.strftime("%-d %B %Y, %l.%M %p") %>, your programme and next steps will change</p>
```

Comment:
I've been formatting the deadline as `"%d/%m/%Y"`. Either way, we should make it consistent 🙂

Reply:
I agree, probably question for Al. In any case we should store the common formatting somewhere - either I18n,  Time::DATE_FORMATS (however this would not work for dates AFAIR) or just a helper (I hate helpers!)

Reply:
Shani is the person here, this is definitely content 🙂

Reply:
It looks like we already have the [GOV.UK date formats](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates) in `config/initializers/date_formats.rb`.

Reply:
@jamesrose - Good find! It looks it only contains date formats though. I'll add some for the time as well (That is after I confirm we want to display time here :P)